models/activity.js

```diff
@@ -0,0 +1,19 @@
+let mongoose = require('mongoose');
+
+let activitySchema = mongoose.Schema({
+  actor: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'User',
+    required: false
```

Comment:
Unsure here if this is correct, there likely should be a consideration for system level activity items - like website blog posts or updates. Those could be tied to a system User though.